src/fineImg.js

```diff
@@ -0,0 +1,24 @@
+const giphy = document.getElementById('giphy');
+
+const changeBackground = (gif) => {
+  giphy.style.background = `url(${gif})`;
+  giphy.style.backgroundSize = 'cover';
+  giphy.style.backgroundRepeat = 'no-repeat';
```

Comment:
Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)

Reply:
The problem is that the background comes from an API so I can't  set that background to a css class